apache/datadog_checks/apache/apache.py

```diff
@@ -110,8 +117,23 @@ def check(self, instance):
                     % instance['apache_status_url']
                 )
 
-    def _collect_metadata(self, value):
-        raw_version = value.split(' ')[0]
-        version = raw_version.split('/')[1]
+        if not version_submitted:
+            # Can't get it from the mod_status output, try to get it from the server header even though
+            # it may not be exposed with some configurations.
+            server_version = r.headers.get("Server")
+            if server_version:
+                self._submit_metadata(server_version)
+
+    def _submit_metadata(self, value):
+        """Possible formats:
+            Apache | Apache/X | Apache/X.Y | Apache/X.Y.Z | Apache/X.Y.Z (<OS>)
```

Comment:
AFAIU you don't handle output 2 and 3.

Reply:
Yes indeed, I'm not sure if it makes sense to submit just the major. Incomplete data can be worse than no data in that case. Open to discussion though

Reply:
That's fine. Leave a comment stating this maybe.